src/HeapDump/HeapDump.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- net451 is actually just another net45 build build with the platform set to x64 -->
+    <TargetFrameworks>net45;net451</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Prefer32Bit>true</Prefer32Bit>
```

Comment:
:memo: This line is unnecessary (set below for 32-bit build, and meaningless for the 64-bit build).